src/toolkits/object_detection/object_detector.cpp

```diff
@@ -493,6 +493,22 @@ void object_detector::perform_predict(gl_sframe data,
   // For each anchor box, we have 4 bbox coords + 1 conf + one-hot class labels
   int num_outputs_per_anchor = 5 + static_cast<int>(class_labels.size());
   int num_output_channels = num_outputs_per_anchor * anchor_boxes().size();
+
+  float_array_map pred_config = get_prediction_config();
+  pred_config["num_classes"] =
+      shared_float_array::wrap(data_iter->class_labels().size());
+  pred_config["lmb_noobj"] = shared_float_array::wrap(5.0);
+  pred_config["lmb_obj"] = shared_float_array::wrap(100.0);
+  pred_config["lmb_coord_xy"] = shared_float_array::wrap(10.0);
+  pred_config["lmb_coord_wh"] = shared_float_array::wrap(10.0);
+  pred_config["lmb_class"] = shared_float_array::wrap(2.0);
```

Comment:
Oh, let's also add a TODO wherever this ends up to make the MPS path actually use these parameters, instead of the versions hardcoded in our MPS code